GitUI/Themes/README.md

```diff
@@ -0,0 +1,68 @@
+﻿# Overview
+
+GitExtensions supports color customization, in particular user can choose between
+default (operating-system-defined), bright or dark theme.
+
+There is a [Wiki page on Dark Theme](https://github.com/gitextensions/gitextensions/wiki/Dark-Theme)
+with some background information and a list of known issues.
+
+# Themes location
+
+Themes are stored in *.css files. Where GitExtensions looks for theme files depends on whether it
+is Installed or Portable version. Both come with a predefined set of themes in
+`(App install)\Themes` directory.
+
+In addition the Installed version looks for user-defined themes in
+`%AppData%\Roaming\GitExtensions\GitExtensions\Themes`
+
+For each found theme file ColorSettings page will show an entry in Theme drop down list.
+
+## Installed GitExtension application
+
+### Official themes
+
+Installed version of GitExtensions stores themes in `(App install)\Themes` directory.
+
+### Custom themes
+
+Any modifications for personal use should be saved to
+`%AppData%\Roaming\GitExtensions\GitExtensions\Themes`, because any changes to
+`(App install)\Themes` directory will be lost as soon as user updates or reinstalls GitExtensions.
```

Comment:
```suggestion
Any modifications for personal use should be saved to
`%AppData%\Roaming\GitExtensions\GitExtensions\Themes` directory, because any changes to
`(App install)\Themes` directory will be lost as soon as user updates or re-installs the app.
```